docs/GettingStartedDocs/buildandsign.md

```diff
@@ -133,3 +132,50 @@ specify a `CONFFILE` that only changed that property:
 # Enclave settings:
 Debug=0
 ```
+
+## Signing an OP-TEE Enclave
+
+The `oesign` tool is currently SGX-only.  For OP-TEE enclaves, signing
+is instead performed by the `sign.py` script that comes with OP-TEE, and the
+script uses a key that is integrated into OP-TEE itself.  The signing key
```

Comment:
Technically, the script doesn't automatically use the key in the TA Dev Kit. When building the SDK from source, it is the CMake `add_enclave` macro that picks up the key from the right place. If users wish to call `sign.py` manually, which they currently have to if consuming the SDK in binary form*, they must still pass the path to the key.

We still don't have official binary releases of the SDK for OP-TEE, though. So perhaps for this document, you can remove the words between the comma and the period:

For OP-TEE enclaves, signing is instead performed by the `sign.py` script that comes with OP-TEE. The signing key and script [...]

Reply:
Done